src/Sylius/Bundle/GridBundle/Renderer/TwigGridRenderer.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\GridBundle\Renderer;
+
+use Sylius\Component\Grid\Definition\Action;
+use Sylius\Component\Grid\Definition\Field;
+use Sylius\Component\Grid\Definition\Filter;
+use Sylius\Component\Grid\Renderer\GridRendererInterface;
+use Sylius\Component\Grid\View\GridView;
+use Sylius\Component\Registry\ServiceRegistryInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class TwigGridRenderer implements GridRendererInterface
+{
+    /**
+     * @var \Twig_Environment
+     */
+    private $twig;
+
+    /**
+     * @var string
+     */
+    private $defaultTemplate;
+
+    /**
+     * @var ServiceRegistryInterface
+     */
+    private $fieldsRegistry;
+
+    /**
+     * @var array
+     */
+    private $actionTemplates;
+
+    /**
+     * @var FormFactoryInterface
+     */
+    private $formFactory;
+
+    /**
+     * @var array
+     */
+    private $filterTemplates;
+
+    /**
+     * @param \Twig_Environment $twig
+     * @param ServiceRegistryInterface $fieldsRegistry
+     * @param string $defaultTemplate
+     * @param array $actionTemplates
+     * @param FormFactoryInterface $formFactory
+     * @param array $filterTemplates
+     */
+    public function __construct(
```

Comment:
There's a misorder between docblock and declaration. Btw. can we order the arguments like:
```php
public function __construct(
  \Twig_Environment $twig,
  ServiceRegistryInterface $fieldsRegistry,
  FormFactoryInterface $formFactory,
  $defaultTemplate, // all templates together
  array $actionTemplates = [], // optional arguments at the end
  array $filterTemplates = []
) {
  // ...
}
```